src/index.js

```diff
@@ -0,0 +1,17 @@
+import './styles.css';
+import nav from './navbar';
+import main from './main/main';
+import footer from './footer';
+
+function component() {
+  const element = document.createElement('div');
+  element.id = 'content';
```

Comment:
This element wasn't meant to be created dynamically from JS. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))